### ISSUE TITLE
Remove a role of a user

### ISSUE THREAD

**Issue by Zorin95670** [ditrit/leto-modelizer-admin#33]
## Description

blocked by #84 

On UserPage, add an action on RolesTable. 
Make actions optional and table don't to do the result of action, only send the event that action are click on the parent.

You can call the props something like that: `with-delete-action`.

On delete event, display a confirmation popup and confirmation delete the role user association.

![image](https://github.com/ditrit/leto-modelizer-admin/assets/6179956/e22d8868-46d4-4c80-b466-b1625315cdcf)
![image](https://github.com/ditrit/leto-modelizer-admin/assets/6179956/29246057-6228-431b-8a8b-c3df99b6b0d8)

**Comment by justinehell** [comment]
Task put on hold, will be done when the new api is up.
